e2e/browser/test/e2e.playwright.ts

```diff
@@ -19,9 +19,69 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-import { test, expect } from "@inrupt/internal-playwright-helpers";
+import { test, expect } from "./fixtures";
 
-test("Granting access to a resource, then revoking it", async ({
+test("Redirect to Podbrowser to accept Access Request", async ({
+  page,
+  auth,
+  accessRequest,
+  idp,
+}) => {
+  // Initial login
+  await auth.login({ allow: true });
+  // The test issues an access request on behalf of a requestor thru the fixture
+  // then the test writes the id to a readable input text input
+  await page.getByTestId("access-request-id").fill(accessRequest);
+
+  // Playwright test reads that access grant id
+  // The test user clicks to be redirected to Podbrowser
+  await Promise.all([
+    page.click("button[data-testid=redirect-for-access]"),
+    page.waitForURL("https://podbrowser.inrupt.com/*"),
```

Comment:
My understanding was that the `click` event should go last, so that all listeners are set up before the event is triggered. Does this make sense, or am I mistaken and the order isn't important?